packages/react/src/components/Notification/Notification.js

```diff
@@ -399,7 +399,7 @@ ToastNotification.propTypes = {
 ToastNotification.defaultProps = {
   kind: 'error',
   title: 'provide a title',
-  caption: 'provide a caption',
+  caption: '',
```

Comment:
Would it make sense to remove the `caption` as a default prop?

Reply:
Yeah, good point!